github/projects.go

```diff
@@ -261,12 +261,15 @@ func (s *ProjectsService) MoveProjectColumn(ctx context.Context, columnID int, o
 //
 // GitHub API docs: https://developer.github.com/v3/repos/projects/
 type ProjectCard struct {
+	URL        *string    `json:"url,omitempty"`
 	ColumnURL  *string    `json:"column_url,omitempty"`
 	ContentURL *string    `json:"content_url,omitempty"`
 	ID         *int       `json:"id,omitempty"`
 	Note       *string    `json:"note,omitempty"`
+	Creator    *User      `json:"creator,omitempty"`
 	CreatedAt  *Timestamp `json:"created_at,omitempty"`
 	UpdatedAt  *Timestamp `json:"updated_at,omitempty"`
+	ColumnID   *int       `json:"column_id,omitempty"`
```

Comment:
Since `ColumnID` will be a field populated in Webhook payloads only, let's be consistent with the pattern in `event_types.go` and separate it out from the rest of the fields with a comment:

```Go
	...
	Creator    *User      `json:"creator,omitempty"`
	CreatedAt  *Timestamp `json:"created_at,omitempty"`
	UpdatedAt  *Timestamp `json:"created_at,omitempty"`

	// The following fields are only populated by Webhook events.
	ColumnID *int `json:"column_id,omitempty"`
}
```